lti_emailer/settings/unit_test.py

```diff
@@ -1,16 +1,13 @@
-from .local import *
+from .base import *
```

Comment:
you're losing ENV_NAME by doing this...do we need to set it to 'unit_test' here?